tests/src/utility/constants.rs

```diff
@@ -94,7 +94,7 @@ pub(crate) const CONTRACT_1_0_0_WASM: &str = "1_0_0/contract.wasm";
 pub(crate) const MINT_1_0_0_WASM: &str = "1_0_0/mint_call.wasm";
 pub(crate) const PAGE_SIZE: u64 = 1000;
 pub(crate) const UNMATCHED_HASH_COUNT: &str = "unmatched_hash_count";
-pub(crate) const PAGE_DICTIONARY_PREFIX: &str = "page-";
+pub(crate) const PAGE_DICTIONARY_PREFIX: &str = "page_";
```

Comment:
Just a detail it's a matter of choice I guess but I would remove the separator from the prefix and replace occurrences of PAGE_DICTIONARY_PREFIX
```
 &format!("{PAGE_DICTIONARY_PREFIX}{SEP}{page_number}"),
 ```
 and 
 
 ```
 format!("{nft_receipt}{SEP}m{SEP}{PAGE_SIZE}{SEP}p{SEP}{page_table_entry}")
 ```
 one day you might want something like that
  ```
 "format!("{PAGE_DICTIONARY_PREFIX}{SEP}{page_table_entry}{SEP}{page_number}")
 ```
 
 When I run make lint I have another rule of formatting apparently.

Reply:
Meh, personally I like having the seperators be in the macro invocation

Reply:
I understand but then consider removing it from the prefix

```
 &format!("{PAGE_DICTIONARY_PREFIX}_{page_number}"),
```

like 

```
pub const HASH_KEY_NAME_PREFIX: &str = "cep78_contract_package_";
pub const ACCESS_KEY_NAME_PREFIX: &str = "cep78_contract_package_access_";
```

Those trailing separators are not super conventional.